CONTRIBUTING.md

```diff
@@ -12,14 +12,14 @@ expert, we can use your help.
 This guide will help you get started. **Do not let this guide intimidate you**.
 It should be considered a map to help you navigate the process.
 
-You may also get help with contributing in the [dev channel][dev], please join
-us!
+You may also get help with contributing in [tokio's Discord server][discord],
+`#tracing-dev` channel. Please join us!
```

Comment:
tiny nit: I _believe_ it's possible to generate a Discord link to a specific channel? this is fine, too, though.